src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -79,6 +81,15 @@ object EasyStageDataset {
   }
 
   def createFileAndFolderSdos(dir: File, parentSDO: String, rights: AccessCategory)(implicit s: Settings): Try[Unit] = {
+    val maybeSha1Map: Try[Map[String, String]] = Try {
+      val sha1File = "manifest-sha1.txt"
+      readFileToString(new File(s.bagitDir, sha1File))
+        .split("\\v+") // split into lines
+        .map(_.split("\\h+")) // split into tokens
+        .filter(!_.isEmpty) // skip empty lines
```

Comment:
Either use `.filterNot(_.isEmpty)` or `.filter(_.nonEmpty)`.
Personally I prefer the latter...